suite/sanity/resize.go

```diff
@@ -41,11 +41,11 @@ func resize(p interface{}) (gravity.TestFunc, error) {
 		g.OK("download installer", g.SetInstaller(cluster.Nodes, cfg.InstallerURL, "install"))
 		g.OK(fmt.Sprintf("install on %d node", param.NodeCount),
 			g.OfflineInstall(cluster.Nodes[0:param.NodeCount], param.InstallParam))
-		g.OK("status", g.Status(cluster.Nodes[0:param.NodeCount]))
+		g.OK("wait for active status", g.WaitForActiveStatus(cluster.Nodes[0:param.NodeCount]))
 		g.OK("time sync", g.CheckTimeSync(cluster.Nodes))
 		g.OK(fmt.Sprintf("expand to %d nodes", param.ToNodes),
 			g.Expand(cluster.Nodes[0:param.NodeCount], cluster.Nodes[param.NodeCount:param.ToNodes],
 				param.InstallParam))
-		g.OK("status", g.Status(cluster.Nodes[0:param.ToNodes]))
+		g.OK("wait for active status", g.WaitForActiveStatus(cluster.Nodes[0:param.ToNodes]))
```

Comment:
nit: `0` is redundant here

Reply:
This was latent in the code already.  As such, I removed it in a separate commit from my other work: f364d2905a21789be0d70ccee346cdb2c34a6534.